chanzuckerberg/idseq-web
# surface informative error if too many pairs are broken

Goes with https://github.com/chanzuckerberg/idseq-dag/pull/171.

# Tests
- Run a samples whose read pairs don't match up and see the pipeline fail gracefully with an informative error.
- Run a normal test sample and see it succeed.

![image](https://user-images.githubusercontent.com/16944816/61836274-7bcbd880-ae34-11e9-8c6a-8eea9ea7918d.png)
